lib/Dialect/FIRRTL/Transforms/LowerIntrinsics.cpp

```diff
@@ -644,7 +661,9 @@ std::pair<const char *, std::function<bool(InstanceGraph &, FModuleLike)>>
         {"circt.mux4cell", lowerCirctMuxCell<false>},
         {"circt_mux4cell", lowerCirctMuxCell<false>},
         {"circt.has_been_reset", lowerCirctHasBeenReset},
-        {"circt_has_been_reset", lowerCirctHasBeenReset}};
+        {"circt_has_been_reset", lowerCirctHasBeenReset},
+	{"fpga_probe", lowerCirctProbe},
+	{"circt.fpga_probe", lowerCirctProbe}};
```

Comment:
Did you mean the following :paperclip:?
```suggestion
	{"circt.fpga_probe", lowerCirctProbe},
	{"circt_fpga_probe", lowerCirctProbe}};
```